### ISSUE TITLE
[docs] New page - Stat component

### ISSUE THREAD

**Issue by marionnegp** [RedHat-UX/red-hat-design-system#404]
**XD link to stat component:** https://xd.adobe.com/view/067ac1e2-5343-4dd7-adf4-ef3e18a91387-5329/

**XD link to stat documentation (WIP):** https://xd.adobe.com/view/a42197ce-a0ab-4f77-ad5c-bfe1368e3950-f172/

- Documentation can follow outline set by [Footer page](https://ux.redhat.com/components/footer/)

**Comment by marionnegp** [comment]
@ronniemcbride @coreyvickery, here's the draft of the stat page content: https://docs.google.com/document/d/1YMWI2FdKZeo-3KN64iAlMpLrK6gIkkK-evlAGmErbn4/edit?usp=sharing.

- I ended up using the sections from the proposed IA. I figured this would be added after the IA updates, and there's more info in the new IA structure.
- When I'd import images from XD, I had some resizing issues, so I dropped in links to specific XD artboards.
